docs/en/couchbase_client.md

```diff
@@ -102,15 +239,13 @@ std::string scope = "my_scope";
 std::string collection = "my_collection";
 
 // Add document to specific scope and collection
-if (couchbase_client.CouchbaseAdd("doc::key", json_data, bucket, scope, collection)) {
-    LOG(INFO) << "Document added to " << bucket << "." << scope << "." << collection;
-}
+couchbase_client.CouchbaseAdd("doc::key", json_data, bucket, scope, collection);
 
 // Get document from specific scope and collection
-auto [success, data] = couchbase_client.CouchbaseGet("doc::key", bucket, scope, collection);
+couchbase_client.CouchbaseGet("doc::key", bucket, scope, collection);
 ```
 
-**Note**: If scope and collection are not specified, they default to `"_default"`.
+**Note**: If scope and collection are not specified, they set to `"_default"`.
```

Comment:
Kind of funny English here to my en_US eyes.  Instead "are set" might be better.